mav_trajectory_generation_ros/PVAJS[] PVAJS_array